event_lead_event/wizard/lead_event.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="event_lead_event_view" model="ir.ui.view">
+        <field name="name">event.event.leadconfirm.form</field>
+        <field name="model">event.event.leadconfirm</field>
+        <field name="arch" type="xml">
+            <form string="Change Lead Event">
+                <group>
+                    <field name="old_lead_id" />
+                    <field name="event_id" />
+                </group>
+                <p>
+                    The current lead event will be replaced with the one specified as new lead event.
+                    If both are the same the lead event will be set to nothing.
```

Comment:
Seriously ? :fearful: 
If both are the same event, shouldn't the wizard just keep this event as lead_event ?

Or maybe, as the wizard is used to remove the lead event, you should use a slightly different view, or use a different message here to show both events are the same and the wizard will do something different ?

Reply:
```python
same_event = fields.Boolean(compute='_compute_same_event')

@api.depends('event_id', 'old_lead_id')
def _compute_same_event(self):
    for item in self:
        item.same_event = item.old_lead_id == item.event_id
```
```xml
<field name="same_event" invisible="1" />
<field name="old_lead_id" invisible="same_event"/>
<div class="alert alert-info" invisible="same_event">
Disable lead status on this event
</div>
<div class="alert alert-warning" invisible="not same_event">
If you confirm you swap bla bla bla...
</div>
```
kind of....